Objects/TI4_Helpers/TI4_UnitHelper.ttslua

```diff
@@ -297,6 +347,33 @@ local _flagshipModifiers = {
             end
         end
     },
+
+    ['Quetzecoatl'] = {
+        description = "Other players cannot use SPACE CANNON against your ships in this system",
+        owner = OWNER.OPPONENT,
+        type = TYPE.MUTATE,
+        apply = function(unitAttrs)
+            for unitType, attr in pairs(unitAttrs) do
+                if attr.spaceCannon then
+                    attr.spaceCannon.dice = 0
+                end
+            end
+        end
+    },
+
+    ["Visz El Vir"] = {
+        description = "Your mechs in this system roll 1 additional die during combat",
+        owner = OWNER.SELF,
+        type = TYPE.ADJUST,
+        apply = function(unitAttrs)
+            for unitType, attr in pairs(unitAttrs) do
+                if unitType == 'Mech' and attr.spaceCombat then
```

Comment:
the "attr.spaceCombat" in the if statement seems odd, I guess only want to buff space combat when present but then the groundCombat should be outside the if block.